test/test_explain_model.py

```diff
@@ -650,6 +658,32 @@ def test_explain_model_pandas_string(self, tabular_explainer):
         assert len(global_explanation.global_importance_values) == num_cols
         assert global_explanation.num_features == num_cols
 
+    def test_create_kernel_explanation_from_interpret_data(self, verify_tabular):
+        interpret_explanation = self.create_kernel_explanation()
+        interpret_explanation_data = interpret_explanation.data()
+        explanation = create_kernel_explanation_from_interpret_data(interpret_explanation_data)
```

Comment:
Should probably test whether that explanation "quacks" with the correct mixins - see test_does_quack or the explanation upload code for examples